src/install/ibm-mq/linux/requirements-tarball.mdx

```diff
@@ -0,0 +1,11 @@
+---
+componentType: default
+headingText: Check the compatibility and requirements
+---
+
+* [Install the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent-new-relic), minimum version 1.24.0.
+* Linux distribution or Windows version [compatible with the infrastructure agent](/docs/infrastructure/new-relic-infrastructure/getting-started/compatibility-requirements-new-relic-infrastructure#operating-systems).
+* [IBM MQ Redistributable Client libraries](https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/messaging/mqdev/redist/) on the agent box.
+* Check or add the libraries path in the `LD_LIBRARY_PATH` environment variable in Linux or follow [these steps](https://www.ibm.com/docs/en/ibm-mq/9.1?topic=windows-testing-communication-between-client-server) on Windows.
```

Comment:
"check or add" doesn't make sense. What are they checking for? What should they see?